types/options.go

```diff
@@ -49,6 +49,12 @@ var (
 	defaultConfigFile = SystemConfigFile
 	// DefaultStoreOptions is a reasonable default set of options.
 	defaultStoreOptions StoreOptions
+
+	// defaultOverrideConfigFile path to override the default system wide storage.conf file
+	defaultOverrideConfigFile = "/etc/containers/storage.conf"
+
+	// defaultDropInConfigDir path to the folder containing drop in config files
+	defaultDropInConfigDir = "/etc/containers/storage.conf.d"
```

Comment:
these may need to remain in the platform specific paths. i'm not sure if there are windows paths that need to be supported.

Reply:
I moved it here because all platform specific files had same value for that variable, including windows one. Although it does look incorrect for the windows, but I just moved it from [options_windows.go](https://github.com/containers/storage/pull/1885/files#diff-fa5a62b5c68d186af67295d75ab2d2978b00b39db884f9828686e68c5ccbbc91)

Reply:
Also, if I don't add it here, then just like `defaultOverrideConfigFile`, I will have to add new `defaultDropInConfigDir` to all platform specific files, otherwise the compilation would fail on those platforms.